### PR TITLE
Ntveplugin 246 merge

### PR DESCRIPTION
**Pull request by justnoxx**
Please, do not merge it yet.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -161,4 +161,4 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
-</project>
```
Comment (ovasilko) [inline pom.xml:164]: @justnoxx can we remove this diff?

File: src/main/java/org/jenkinsci/plugins/electricflow/ElectricFlowPipelinePublisher.java [hunk 2]
```diff
@@ -268,7 +268,7 @@ public String getStoredPipelineName() {
 
   private JSONArray getPipelineParameters() {
 
-    if (addParam != null && !addParam.isEmpty() && !"{}".equals(addParam)) {
```
Comment (ovasilko) [inline src/main/java/org/jenkinsci/plugins/electricflow/ElectricFlowPipelinePublisher.java:271]: it seems to be some fix done within the branch

File: src/main/java/org/jenkinsci/plugins/electricflow/ElectricFlowPublishApplication.java [hunk 3]
```diff
@@ -78,7 +78,7 @@ public ElectricFlowPublishApplication(String configuration, String filePath) {
   // ~ Methods ----------------------------------------------------------------
 
   public static FilePath makeApplicationArchive(
-      Run<?, ?> build, TaskListener listener, FilePath workspace, String filePath)
```
Comment (ovasilko) [inline src/main/java/org/jenkinsci/plugins/electricflow/ElectricFlowPublishApplication.java:81]: @justnoxx , not sure why this had been added previously in the branch. So I left all this things as it was in the branch - Run<?, ?>

File: src/main/java/org/jenkinsci/plugins/electricflow/ElectricFlowTriggerRelease.java [hunk 4]
```diff
@@ -150,30 +146,17 @@ public void perform(
       run.addAction(action);
       run.save();
       logger.println("TriggerRelease  result: " + formatJsonOutput(releaseResult));
-    } catch (IOException | InterruptedException e) {
```
Comment (ovasilko) [inline src/main/java/org/jenkinsci/plugins/electricflow/ElectricFlowTriggerRelease.java:153]: @justnoxx handling of these both exceptions seems to be similar, so we can use pipe here

File: pom.xml (hunk 1 above)
Comment (justnoxx) [reply pom.xml]: Resolved

File: src/test/java/org/jenkinsci/plugins/electricflow/ElectricFlowAssociateBuildToReleaseTest.java [hunk 5]
```diff
@@ -112,7 +112,7 @@ private void applyFlowConfiguration(String configName) {
             FLOW_USER,
             Secret.fromString(FLOW_PASSWORD),
             FLOW_REST_API_URI_PATH,
-            true);
```
[the remark this answers is not in the document]
Comment (justnoxx) [reply src/test/java/org/jenkinsci/plugins/electricflow/ElectricFlowAssociateBuildToReleaseTest.java:115]: This is tests folder, I did not format them.

File: src/main/java/org/jenkinsci/plugins/electricflow/ElectricFlowPipelinePublisher.java (hunk 2 above)
Comment (justnoxx) [reply src/main/java/org/jenkinsci/plugins/electricflow/ElectricFlowPipelinePublisher.java:271]: That is true. Re-applied fix back.

File: src/main/java/org/jenkinsci/plugins/electricflow/ElectricFlowPublishApplication.java [hunk 6]
```diff
@@ -146,7 +146,6 @@ private boolean runProcess(
       @Nonnull Run<?, ?> run, @Nonnull TaskListener taskListener, @Nonnull FilePath workspace) {
     PrintStream logger = taskListener.getLogger();
 
-    @SuppressWarnings("WrapperTypeMayBePrimitive")
```
Comment (ovasilko) [inline src/main/java/org/jenkinsci/plugins/electricflow/ElectricFlowPublishApplication.java]: @justnoxx this change was in branch NTVEPLUGIN-209

File: src/test/java/org/jenkinsci/plugins/electricflow/ElectricFlowAssociateBuildToReleaseTest.java (hunk 5 above)
[the remark this answers is not in the document]
Comment (ovasilko) [reply src/test/java/org/jenkinsci/plugins/electricflow/ElectricFlowAssociateBuildToReleaseTest.java:115]: Lets format all *.java files in both main and test folders under src folder

**Comment by ovasilko** [comment]
@justnoxx , there are some conflicts according to merged PR https://github.com/jenkinsci/electricflow-plugin/pull/63 . Could you please check?

**Pull request merged**